.cicd/Jenkinsfile

```diff
@@ -122,17 +122,26 @@ pipeline {
                         }
                     }
 
-                    // Run the unified build script; if successful create a tarball of the build and upload to S3
-                    stage('Build') {
-                        steps {
-                            echo "Building SRW (${env.SRW_COMPILER}) on ${env.SRW_PLATFORM} (using ${env.WORKSPACE})"
-                            sh 'bash --login "${WORKSPACE}/.cicd/scripts/srw_build.sh"'
+                    parallel{
```

Comment:
The workflow failed at line 125.  Looking at the documentation for parallel stages, it would look like:

```
stage('Functional Tests and Build')
       steps {
              parallel(
                    Functional Tests: {
                           echo "Running unittest on retrieve_data.py"
                           sh 'bash --login "${WORKSPACE}/.cicd/scripts/srw_unittest.sh"'
                    },
                    Build: {
                            echo "Building SRW (${env.SRW_COMPILER}) on ${env.SRW_PLATFORM} (using ${env.WORKSPACE})"
                             sh 'bash --login "${WORKSPACE}/.cicd/scripts/srw_build.sh"'
                    }
              )
       }      
}
```

I'm not sure where to put the `post {` part of the build stage.

It might be best to remove the `parallel { ` on line 125 and just have a three part pipeline.